### PR TITLE
s-salome-tar: Remove HELLO PYHELLO from IHM

### PR DESCRIPTION
**Pull request by estreby**
Checked from installer

### CODE REVIEW

File: scripts.d/s-salome-tar [hunk 1]
```diff
@@ -133,6 +133,12 @@ function s-salome-tar-install()
         done
     fi
 
+    #  Do clean-up HELLO and PYHELLO for the release
+    for M in HELLO PYHELLO; do
+        sed -i -E "s&^.*SalomeAppConfig.*/BINARIES-DB10/$M/share/salome/resources/&#\0&g" \
```
Comment (TurboGit) [inline scripts.d/s-salome-tar:138]: Hum... This is obviously wrong and will not work on DB11 or DB09. I would just remove this part, the match would still be unambiguous I think. Also please use `,` and not `&` as sed separator to be consistent with current sed commands.

So I propose:

`"s,^.*SalomeAppConfig.*/$M/share/salome/resources/,#\0,g"`

Please double check that it is working fine.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Changes proposed.

File: scripts.d/s-salome-tar [hunk 2]
```diff
@@ -133,6 +133,12 @@ function s-salome-tar-install()
         done
     fi
 
+    #  Do clean-up HELLO and PYHELLO for the release
```
Comment (TurboGit) [inline scripts.d/s-salome-tar:136]: The comment should be:
`# Do remove HELLO and PYHELLO from the UI (no icon`)
and a space after this line to separate from the code.

**Comment by estreby** [comment]
Update with the changes requested and validated by non regression test

**Review by TurboGit — COMMENTED** [review]
Works for me. I'll add the missing empty line after the group comment.

**Pull request merged**